ci/tasks/prepare-e2e-codebuild.yml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Could we have a comment block to say what the purpose of this task is? As in, what does preparing an e2e codebuild entail.

Reply:
Sure

Reply:
Nice, thanks 👍